src/content/reference/cluster-upgrades/_index.md

```diff
@@ -41,44 +41,62 @@ Among the third party components building a tenant cluster stack are
+At Giant Swarm we bundle specific versions of these components of the workload cluster stack into a **workload cluster release**. A workload cluster release is specific for a provider (AWS, Azure, or KVM) and identified by a version number. To learn more about workload cluster releases and our versioning, check the [workload cluster releases reference](/reference/tenant-cluster-release-versions/).
 
-Once deployed, the tenant cluster stack is **immutable**.
+Once deployed, the workload cluster stack is **immutable**.
 All components are deployed based on images, either of virtual machines or of Docker containers.
 No changes are ever made to components at runtime.
 This ensures that the stack running in your environment is the exact stack we have tested before.
 
-As a consequence, the only way to change the stack is to perform an upgrade, to switch to a new tenant cluster release.
+As a consequence, the only way to change the stack is to perform an upgrade, to switch to a new workload cluster release.
 
 ### Upgrade semantics {#semantics}
 
-According to our tenant cluster release versioning, three different levels of upgrades can occur:
+According to our workload cluster release versioning, three different levels of upgrades can occur:
 
 - *Patch upgrade*: Only the patch version number is increased.
 
 - *Minor upgrade*: The minor version number is increased, while the Major version number stays the same.
 
 - *Major upgrade*: The Major version number is increased.
 
-**Both patch and minor upgrades** can be rolled out at any time by Giant Swarm without your interaction. Currently, this happens in coordination with your administrators and with a notice to your developers.
+#### Patch upgrade {#patch-upgrade}
 
-When a tenant cluster release with a **new major version** becomes available, we inform you, but leave scheduling of the upgrade to you. This gives you the control to decide if and when it is time for you to upgrade, potentially updating workloads first. These upgrades are also accompanied or even triggered by Giant Swarm staff, to ensure we have a close eye on the upgrade process and the uptime of your workloads.
+Patches contain fixes only, something that needs to be done to get rid of a bug. Individual patches are as minimal as possible and they are needed to increase the stability of the cluster or fix a CVE. They are released based on upstream information and/or learnings through any of our customers.
+
+#### Minor upgrade {#minor-upgrade}
+
+In case we are releasing features in a non disruptive way, feature flagged, we will put those in minor releases. Those will actually have even less impact than a patch as they will not change anything that is in place if the customer no explicitly enable them.
```

Comment:
```suggestion
In case we are releasing features in a nondisruptive way (i.e. feature flagged) we will put those in minor releases. Those will actually have even less impact than a patch as they will not change anything that is in place if the customer doesn't explicitly enable it.
```